### PR TITLE
Add IPC-June 2025

### PR DESCRIPTION
**Pull request by Husameldin92**
Could you please merge the IPC Berlin - June 2025. Thank you in advance.

### CODE REVIEW

File: archive/entries/2024-12-13-1.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>International PHP Conference Berlin 2025</title>
+  <id>https://www.php.net/archive/2024.php#2024-12-13-1</id>
+  <published>2024-12-13T11:11:55+01:00</published>
+  <updated>2024-12-13T11:11:55+01:00</updated>
+  <link href="https://www.php.net/conferences/index.php#2024-12-13-1" rel="alternate" type="text/html"/>
+  <link href="https://phpconference.com/berlin/" rel="via" type="text/html"/>
+  <default:finalTeaserDate xmlns="http://php.net/ns/news">2024-11-15</default:finalTeaserDate>
+  <category term="conferences" label="Conference announcement"/>
+  <default:newsImage xmlns="http://php.net/ns/news" link="https://phpconference.com/berlin/" title="International PHP Conference Berlin 2025">IPC_24_Logo.png</default:newsImage>
```
Comment (saundefined) [inline archive/entries/2024-12-13-1.xml:11]: Hi, thanks!

As you use `IPC_24_Logo.png`, could you remove `image (31).png`?

File: archive/entries/2024-12-13-1.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>International PHP Conference Berlin 2025</title>
+  <id>https://www.php.net/archive/2024.php#2024-12-13-1</id>
+  <published>2024-12-13T11:11:55+01:00</published>
+  <updated>2024-12-13T11:11:55+01:00</updated>
+  <link href="https://www.php.net/conferences/index.php#2024-12-13-1" rel="alternate" type="text/html"/>
+  <link href="https://phpconference.com/berlin/" rel="via" type="text/html"/>
+  <default:finalTeaserDate xmlns="http://php.net/ns/news">2024-11-15</default:finalTeaserDate>
```
Comment (sy-records) [inline archive/entries/2024-12-13-1.xml:9]: Wrong time here?

```suggestion
  <default:finalTeaserDate xmlns="http://php.net/ns/news">2024-11-15</default:finalTeaserDate>
```

File: archive/entries/2024-12-13-1.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>International PHP Conference Berlin 2025</title>
+  <id>https://www.php.net/archive/2024.php#2024-12-13-1</id>
+  <published>2024-12-13T11:11:55+01:00</published>
+  <updated>2024-12-13T11:11:55+01:00</updated>
+  <link href="https://www.php.net/conferences/index.php#2024-12-13-1" rel="alternate" type="text/html"/>
+  <link href="https://phpconference.com/berlin/" rel="via" type="text/html"/>
```
[the remark this answers is not in the document]
Comment (Husameldin92) [reply archive/entries/2024-12-13-1.xml:8]: Hello, thanks for the quick reply. The link has been changed

Comment (saundefined) [inline archive/entries/2024-12-13-1.xml:8]:
```suggestion
  <link href="https://phpconference.com/berlin-en/" rel="via" type="text/html"/>
```

Comment (sy-records) [inline archive/entries/2024-12-13-1.xml:8]:
```suggestion
  <link href="https://phpconference.com/berlin-en/" rel="via" type="text/html"/>
```

[the remark this answers is not in the document]
Comment (saundefined) [reply archive/entries/2024-12-13-1.xml:8]: Yet another one :)

**Review by sy-records — CHANGES_REQUESTED** [review]

**Review by saundefined — APPROVED** [review]
LGTM

**Review by sy-records — APPROVED** [review]

**Comment by Husameldin92** [comment]
Thank you so much

**Pull request merged**